src/smt/smt_engine.cpp

```diff
@@ -509,7 +489,7 @@ class SmtEnginePrivate : public NodeManagerListener {
   std::vector<Node> d_boolVars;
 
   /** Assertions in the preprocessing pipeline */
-  AssertionPipeline d_assertions;
+  preproc::AssertionPipeline d_assertions;
```

Comment:
I believe that the `preproc::` here is not necessary since you have a `using namespace CVC4::preproc` above.